src/main/java/org/example/service/ConnectionService.java

```diff
@@ -0,0 +1,57 @@
+package org.example.service;
+
+import java.sql.*;
+
+
+public class ConnectionService {
+
+    private static final String DATABASE_NAME = "postgres";
```

Comment:
Where the `DATABASE_NAME` variable is used?

Reply:
fixed